api/v1alpha1/scheduledscaling_types.go

```diff
@@ -0,0 +1,166 @@
+/*
+MIT License
+
+Copyright (c) 2023 mercari
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ScheduledScalingSpec defines the desired state of ScheduledScaling
+type ScheduledScalingSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	/// The schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
+	Schedule Schedule `json:"schedule" protobuf:"bytes,1,name=schedule"`
+
+	//TargetRef is the targets that need to be scheduled
```

Comment:
nit: Please put space between `//` and your comment.

```suggestion
	// TargetRef is the targets that need to be scheduled
```

In golang, `//hoge` and `// hoge` is completely two different stuff. The former is called `Directive comments`.
Golang automatically generates the document based on the comments like [this](https://pkg.go.dev/github.com/mercari/tortoise@v0.7.9/api/v1beta3) (see [the doc](https://tip.golang.org/doc/comment)), but at that time, directive comments are ignored.
So, we must make sure to use `// hoge` basically for this kind of normal comments describing fields, global variables, functions, etc.